### DISCUSSION TITLE
What makes you contribute more than once to a repo?

### DISCUSSION THREAD

**Discussion by EmmaDawsonDev** [EddieHubCommunity/support#7395] [General]
I would love to know what keeps you coming back to contribute on a repo regularly? 

Also, the opposite- what makes you move on after 1 contribution?

**Comment by Yushi5058** [comment]
I want to say what will eventually keep me coming back to contribute on a repo regularly is the theme itself and the goals of the project, and well-written documentation aimed to help beginners to contribute even if it's a little. What pushes me away may be the tech stack or less beginner-friendly documentation.

**Reply by EmmaDawsonDev** [reply]
So you're saying that if it's something you find interesting and align with the goals/vision you're more likely to continue contributing? 

Is there something that maintainers could do to make that more likely? 

What kind of recognition do you appreciate for your contributions?

**Reply by Yushi5058** [reply]
Yes absolutely! 
Maintainers can make the documentation more accessible and wait for folks who have a common interest in the project's goal. A simple thanks for my contributions is enough and sufficient, I do this not to seek recognition but to contribute for the common good.

**Reply by Yushi5058** [reply]
and because I enjoy contributing too !!

**Comment by eddiejaoude** [comment]
Q. Is there anything that some repos do that you think are great ideas?

For example:
- regular office hour calls?
- always having `good first issues`
- automation to provide feedback sooner
... anything else?

**Comment by MitAbhay** [comment]
Hey @EmmaDawsonDev, I think I keep coming back to contribute to a repo regularly for a few reasons. Firstly, my interest in the project grows with time as I make contributions and strive to make it better. Secondly, contributing regularly allows me to stay in contact with other great developers, which I find enjoyable. Finally, the recognition and feedback I receive for my contributions are very motivating. When others use my work and provide positive feedback, it encourages me to work harder to make things even better.

**Comment by Panquesito7** [comment]
Here are a few reasons (from personal experience) why I contribute more than once to a repository:

1. Usually, if there's anything I can fix or improve, I come back and create a new issue/discussion, and if possible, implement it.
2. You learn on each pull request and issue you create, thus, creating a new experience.
3. There might be great or new opportunities: you never know what else you can contribute to!
4. Meeting new and great folks as well as getting more involved in the project/community.
